index.js

```diff
@@ -1,7 +1,6 @@
 'use strict';
 
-// TODO: get rid of '.Claim;' at the end of the require statement
-const Claim = require('my-lib-54357435').Claim;
+const { Claim } = require('my-lib-54357435');
```

Comment:
Would it be possible to keep the `security` namespace and import this?
```
const { security } = require('my-lib-54357435');
const claim = new security.Claim("tenantId", "123");
```
Not sure if this variant is really better I'm trying to understand...

Reply:
Might be WebStorm related issue. VS Code works fine and even has some nice IntelliSense here. So just ignore this.